Index: WorkersAnalyzer/Test.py
===================================================================
diff --git a/WorkersAnalyzer/Test.py b/WorkersAnalyzer/Test.py
deleted file mode 100644
--- a/WorkersAnalyzer/Test.py	(revision fd6f06dc960353d1d9549cf3e8c3cc917534ae1e)
+++ /dev/null	(revision fd6f06dc960353d1d9549cf3e8c3cc917534ae1e)
@@ -1,15 +0,0 @@
-import pandas as pd
-
-from Core import PDFBlock
-from Extractors.PisaExtractor import PisaExtractor as ExtractorType
-
-filename = "Andreotti Valeria_AOUP_timbrature.pdf"
-
-
-block = PDFBlock.from_file(filename)
-extractor = ExtractorType( block.pages  )
-
-features: list[pd.DataFrame] = extractor.features()
-
-concatenated = pd.concat( features)
-print(concatenated['turno'].value_counts())
\ No newline at end of file
